src/pages/Dashboard/Modals/SupplyWithdraw/SupplyWithdrawForm.tsx

```diff
@@ -0,0 +1,202 @@
+import { useStyles } from '../styles';
+
+interface ISupplyWithdrawFormProps {
+  asset: Asset;
+  tokenInfo: ILabeledInlineContentProps[];
+  userTotalBorrowBalance: BigNumber;
+  userTotalBorrowLimit: BigNumber;
+  dailyEarnings: BigNumber;
+  onSubmit: IAmountFormProps['onSubmit'];
+  enabledButtonKey: string;
+  disabledButtonKey: string;
+  maxInput: BigNumber;
+  calculateNewBalance: (amount: BigNumber) => BigNumber;
+}
+
+export const SupplyWithdrawContent: React.FC<
+  Omit<ISupplyWithdrawFormProps, 'onSubmit'> & FormikProps<FormValues>
+> = ({
+  values,
+  setFieldValue,
+  asset,
+  tokenInfo,
+  userTotalBorrowBalance,
+  userTotalBorrowLimit,
+  dailyEarnings,
+  maxInput,
+  enabledButtonKey,
+  disabledButtonKey,
+  calculateNewBalance,
+}) => {
+  const { id: assetId } = asset;
+  const { amount } = values;
+  const validAmount = amount && !amount.isZero() && !amount.isNaN();
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+  const [newBorrowLimit, setNewBorrowLimit] = useState<BigNumber | undefined>(undefined);
+  const [newBorrowPercent, setNewBorrowPercent] = useState(
+    userTotalBorrowBalance.div(userTotalBorrowLimit).times(100),
+  );
+  useEffect(() => {
+    const tokenPrice = getBigNumber(asset?.tokenPrice);
+    const collateralFactor = getBigNumber(asset?.collateralFactor);
+
+    if (tokenPrice && validAmount) {
+      const amountInUsd = convertWeiToCoins({
+        value: amount as BigNumber,
+        tokenSymbol: asset.id,
+      })
+        .times(tokenPrice)
+        .times(collateralFactor);
+      const temp = calculateNewBalance(amountInUsd);
+      setNewBorrowLimit(BigNumber.maximum(temp, 0));
+      setNewBorrowPercent(userTotalBorrowBalance.div(temp).times(100));
+    } else if (BigNumber.isBigNumber(userTotalBorrowLimit)) {
+      setNewBorrowLimit(undefined);
+      setNewBorrowPercent(userTotalBorrowBalance.div(userTotalBorrowLimit).times(100));
+    }
+  }, [amount, asset?.id, userTotalBorrowBalance, userTotalBorrowLimit]);
+
+  // Convert wallet/ supply balance into wei
+  const maxInputWei = React.useMemo(
+    () => convertCoinsToWei({ value: maxInput, tokenSymbol: assetId }),
+    [maxInput.toString(), assetId],
+  );
+  return (
+    <>
+      <TokenTextField
+        name="amount"
+        tokenSymbol={assetId as TokenId}
+        value={amount}
+        onChange={amt => setFieldValue('amount', amt, true)}
+        maxWei={maxInputWei}
+        rightMaxButtonLabel={t('supplyWithdraw.max').toUpperCase()}
+        css={styles.input}
+      />
+      <Typography component="span" variant="small1" css={styles.greyLabel}>
+        <Trans i18nKey="supplyWithdraw.walletBalance">
+          <Typography component="span" variant="small1" css={styles.whiteLabel}>
+            {{ amount: asset.walletBalance, symbol: assetId?.toUpperCase() }}
+            {asset.walletBalance} {assetId?.toUpperCase()}
+          </Typography>
+        </Trans>
+      </Typography>
```

Comment:
Same thing here the comment disappeared.

You need to add the actual text inside the `Trans` tag:
```jsx
<Typography component="span" variant="small1" css={styles.greyLabel}>
  <Trans i18nKey="supplyWithdraw.walletBalance">
    Wallet balance:{' '}
    <Typography component="span" variant="small1" css={styles.whiteLabel}>
      {amount} {symbol}
    </Typography>
  </Trans>
</Typography>;
```

Reply:
The duplicate value references do need to be removed.